### PR TITLE
[CPF-4069] Call super equals explicitly

### PR DESCRIPTION
**Pull request by KernelDeimos**

### CODE REVIEW

File: src/foam/java/refinements.js [hunk 1]
```diff
@@ -556,6 +556,14 @@ foam.LIB({
               return `get${foam.String.capitalize(p.name)}();`
             }).join('\n')
         });
+
+        cls.method({
+          name: 'equals',
+          type: 'boolean',
+          args: [ { name: 'o', type: 'Object' } ],
+          visibility: 'public',
+          body: `return super.equals(o);`
```
Comment (kgrgreer) [inline src/foam/java/refinements.js:565]: I don't like this solution. Better to just re-insert super code here to be sure it's correct.

Comment (kgrgreer) [reply src/foam/java/refinements.js:565]: This is just cheating.

Comment (KernelDeimos) [reply src/foam/java/refinements.js:565]: You mean move the method out of AbstractFObject and put it in refinements? I would've done that but I wasn't sure if some models manually override equals

Comment (kgrgreer) [reply src/foam/java/refinements.js:565]: No, just copy the code out of the super class and put it here in body:

**Comment by KernelDeimos** [comment]
Closing this because it can be fixed better by moving `hashCode` instead

**Pull request closed without merging**